schema.graphql

```diff
@@ -59,6 +59,8 @@ type Geyser @entity {
   funded: BigDecimal!
   distributed: BigDecimal!
   gysrSpent: BigDecimal!
+  sharesPerSecond: BigDecimal!
+  fundings: [Funding!]!
```

Comment:
might need to maintain a `sharesPerToken` for both the staking and reward side now

Reply:
I'm going to save this for when we start bringing in changes for the v2 contracts

Reply:
This actually isn't related to v2, just will need it client side to easily convert funding shares into amounts in the UI